src/components/Account/AccountProfileContainer.tsx

```diff
@@ -252,15 +252,15 @@ class AccountProfileContainer extends React.Component<IProps, null> {
                     <div className={css.otherInfoContainer}>
                       <div className={css.tokens}>
                         {accountInfo
-                           ? <div><strong>Rep. Score</strong><br/><ReputationView reputation={accountInfo.reputation} totalReputation={dao.reputationTotalSupply} daoName={dao.name}/> </div>
-                           : ""}
-                         <div><strong>GEN:</strong><br/><span>{formatTokens(genBalance)}</span></div>
--                        <div><strong>ETH:</strong><br/><span>{formatTokens(ethBalance)}</span></div>
+                          ? <div><strong>Rep. Score</strong><br /><ReputationView reputation={accountInfo.reputation} totalReputation={dao.reputationTotalSupply} daoName={dao.name} /> </div>
+                          : ""}
+                        <div><strong>GEN:</strong><br /><span>{formatTokens(genBalance)}</span></div>
+                        -                        <div><strong>ETH:</strong><br /><span>{formatTokens(ethBalance)}</span></div>
```

Comment:
some weird whitespace here

Reply:
It seems that my editor adds those.  I'm not sure it is from the linting rules or what.  But those spaces are not a problem.

Reply:
This appears to explain reasoning behind adding the space:  https://stackoverflow.com/questions/462741/space-before-closing-slash

Reply:
the whitespace linting rule itself is not a problem either way, but you PR becomes hard to read if you use different linting rules (and "fix" them) then we do, which _is_ important to us :-) So if you can get your linter in agreement;  or do not commit these type of fixes, life would be better :-)